CHANGES.md

```diff
@@ -5,7 +5,7 @@ Release Notes.
 8.7.0
 ------------------
 #### Project
-
+* Supports modifying span attributes in async mode.
```

Comment:
Codes are good, this is an agent change, please fix changelog.

Reply:
Done